discovery/azure/azure.go

```diff
@@ -45,6 +46,7 @@ const (
 	azureLabelMachineLocation      = azureLabel + "machine_location"
 	azureLabelMachinePrivateIP     = azureLabel + "machine_private_ip"
 	azureLabelMachineTag           = azureLabel + "machine_tag_"
+	azureLabelMachineScaleSet      = azureLabel + "machine_scale_set"
```

Comment:
This would need to be documented in `docs/configuration/configuration.md`

Reply:
I will create an PR in https://github.com/prometheus/docs, thanks for the hint!

Reply:
It's all in this repo these days, and should be in this PR

Reply:
Thanks for the hint, I added it.